sdk/monitor/azure-monitor-query/src/test/java/com/azure/monitor/query/LogsQueryAsyncClientTest.java

```diff
@@ -9,8 +9,8 @@
 import com.azure.core.http.HttpClient;
 import com.azure.core.http.policy.RetryPolicy;
 import com.azure.core.http.policy.RetryStrategy;
-import com.azure.core.test.TestBase;
 import com.azure.core.test.TestMode;
```

Comment:
Should we update testLogsQueryAllowPartialSuccess & testServerTimeout to DoNotRecord?